git/install_hooks.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+curl -o .github/workflows/pull-request-lint.yml \
```

Comment:
```suggestion
curl -o git/workflows/pull-request-lint.yml \
```
ask; 뭔가 이런걸 의도하셨을까요?